src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null
+                && email.equals(UserService.findByEmail(email).getEmail())
+                && password.equals(UserService.findByEmail(email).getPassword())) {
```

Comment:
let's save the User that we got from `findByEmail()` method, into a local variable